reporter/html.js

```diff
@@ -723,6 +727,10 @@ QUnit.testDone(function( details ) {
 		time.innerHTML = details.runtime + " ms";
 		testItem.insertBefore( time, assertList );
 	}
+
+	sourceName.innerHTML = "<b class='counts'>Source: </b>" + details.source;
+	assertList.parentNode.insertBefore(sourceName, assertList.nextSibling);
```

Comment:
@leobalter I tried the above. But gives error for addEvent because it calls addevent two time for testTitle click. 
Changed  innerhtml  to strong and used appendChild instead of addBefore.
Seems to be working fine now.